### DISCUSSION TITLE
ggplotly tidak interaktif di RSTUDIO

### DISCUSSION THREAD

**Discussion by ghost** [teamalgoritma/community#81] [Q&A] [answered]
apakah normal jika kita sudah ggploly tapi di rstudionya tidak interaktif tapi pada saat di knit interaktif? mengapa bisa begitu apa karena ram? karena saya rasa harusnya ram saya tidak begitu kecil

**Comment by NabiilahArdini** [comment, marked as answer]
Halo Bapak @EdbertHalim 

Umumnya tidak seperti itu. Namun saya baca di diskusi [ini](https://stackoverflow.com/questions/71496184/greyed-out-and-non-interactive-preview-of-plotly-in-r-markdown-in-rstudio), sepertinya terdapat bug untuk preview plotly di RStudio. Namun kabar terbaru sepertinya sedang/sudah diselesaikan dan akan diupdate di release berikutnya. Semoga bug ini sudah dapat diperbaiki pada versi terbaru.

**Reply by ghost** [reply]
Ooo baik terimakasih infonya
